### PR TITLE
Fix license in Cargo.toml files

### PR DESCRIPTION
**Pull request by mati865**
Missed in https://github.com/rust-lang/rust-clippy/pull/3269

changelog: none

r? @Manishearth

### CODE REVIEW

**Comment by mati865** [comment]
Right, forgot the CI is broken.

**Comment by Manishearth** [comment]
Huh, why's it broken? Do we have an issue for this?

**Comment by mati865** [comment]
There is no issue opened but on every PR (since today?) fails with:
```
error: no such subcommand: `+nightly`
```

To me it looks like `cargo` command isn't proxied through rustup wrappers but runs cargo directly. I suppose it could be AppVeyor issue but I haven't investigated.

File: clippy_lints/Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ authors = [
 description = "A bunch of helpful lints to avoid common pitfalls in Rust"
 repository = "https://github.com/rust-lang/rust-clippy"
 readme = "README.md"
-license = "MPL-2.0"
+license = "MIT/Apache-2.0"
```
Comment (lzutao) [inline clippy_lints/Cargo.toml]:
```suggestion
license = "MIT OR Apache-2.0"
```

File: mini-macro/Cargo.toml [hunk 2]
```diff
@@ -8,7 +8,7 @@ authors = [
 	"Martin Carton <cartonmartin@gmail.com>",
 	"Oliver Schneider <clippy-iethah7aipeen8neex1a@oli-obk.de>"
 ]
-license = "MPL-2.0"
+license = "MIT/Apache-2.0"
```
Comment (lzutao) [inline mini-macro/Cargo.toml]:
```suggestion
license = "MIT OR Apache-2.0"
```

**Comment by mati865** [comment]
> 
> 
> It is recommended to use OR operator instead: [doc.rust-lang.org/cargo/reference/manifest.html#package-metadata](https://doc.rust-lang.org/cargo/reference/manifest.html#package-metadata)

I just copied it from topmost `Cargo.toml`: https://github.com/rust-lang/rust-clippy/blob/737f0a6bb508706b75e21194e3010aa3865e779a/Cargo.toml#L14
I'll update it later or tomorrow.

**Comment by mati865** [comment]
Ready for review.

**Comment by mati865** [comment]
@Manishearth it won't  work until https://github.com/rust-lang/rust-clippy/pull/4628 is merged.

I think Clippy could do `@bors treeclose` whether the CI is broken and reopen it when rustups are merged.

**Comment by mati865** [comment]
Let's merge it before Clippy breaks again.

**Pull request merged**